src/gc.h

```diff
@@ -313,15 +313,15 @@ STATIC_INLINE struct jl_gc_metadata_ext page_metadata_ext(void *_data)
     i = REGION_INDEX(data);
     info.pagetable_i = i % 32;
     info.pagetable_i32 = i / 32;
-    info.pagetable1 = memory_map.meta1[i];
+    info.pagetable1 = jl_assume(memory_map.meta1[i]);
     i = REGION1_INDEX(data);
     info.pagetable1_i = i % 32;
     info.pagetable1_i32 = i / 32;
-    info.pagetable0 = info.pagetable1->meta0[i];
+    info.pagetable0 = jl_assume(info.pagetable1->meta0[i]);
     i = REGION0_INDEX(data);
     info.pagetable0_i = i % 32;
     info.pagetable0_i32 = i / 32;
-    info.meta = info.pagetable0->meta[i];
+    info.meta = jl_assume(info.pagetable0->meta[i]);
     assert(info.meta);
```

Comment:
True. These two were added while I was trying around. Shouldn't be necessary anyway now. OTOH, I was hoping that the compiler will generate assertion in debug mode although I didn't check.......